### ISSUE TITLE
Accessibility - High Contrast

### ISSUE THREAD

**Issue by SteveLindsay461** [MHRA/products#195]
**Describe the bug**
Using ‘High contrast’, a common Chrome plugin, the site does not seem to remember the preference when going to another page.
Turning the plugin on works, but when navigating to any other page or performing a search, the site then reverts back to standard contrast.
This does not happen on other websites checked

**Screenshots**
![image](https://user-images.githubusercontent.com/58432596/70813391-4adb4880-1dc1-11ea-9dfb-7780c7bba335.png)

When clicking to any next page, it turns off
![image](https://user-images.githubusercontent.com/58432596/70813436-5c245500-1dc1-11ea-9815-d77e33aeae81.png)

**Additional context**

**Comment by craiga** [comment]
FWIW, the Chrome Web Store itself doesn't work with this extension either.
